src/components/common/range-brush.js

```diff
@@ -120,9 +120,12 @@ export default class RangeBrush extends Component {
   }
 
   render() {
-    return <StyledG className="kg-range-slider__brush"
-                    innerRef={comp => {
-      this.rootContainer = comp;
-    }}/>;
+    return (
+      <StyledG
+        className="kg-range-slider__brush"
+        innerRef={comp => {
```

Comment:
`innerRef` has been deprecated in `styled-components` v4 we could just use ref now.

https://www.styled-components.com/docs/api#deprecated-innerref-prop